packages/gatsby-theme-docs/package.json

```diff
@@ -76,7 +77,13 @@
     "react-router-dom": "5.3.0",
     "rehype-slug": "4.0.1",
     "remark-emoji": "2.2.0",
-    "slugify": "1.6.5"
+    "remark-frontmatter": "3.0.0",
+    "remark-parse": "9.0.0",
+    "remark-rehype": "8.1.0",
+    "slugify": "1.6.5",
+    "unified": "9.2.2",
+    "unist-util-filter": "2.0.3",
+    "unist-util-visit": "2.0.3"
```

Comment:
TBD understand why these are all added, only unist-util-visit is used in the new code.  I think it's a fix for warnings that should have been done anyways but maybe this is rather an artifact or bug.

Reply:
fixed, the dependencies were forgotten to be removed.